### DISCUSSION TITLE
Discord_rpc Not recognizable in the libraries.

### DISCUSSION THREAD

**Discussion by Cheemsandfriends** [FunkinCrew/Funkin#1721] [Q&A]
Hello, So When I try to add my Discord Rich presence I can't. It just appears as it can be detected.
![image](https://user-images.githubusercontent.com/84867412/131790750-048bf019-bd08-45f2-af3c-ce8ce0b9efc9.png)

(This doesn't affect me on compiling. the discord import doesn't want to work, idk why lol)

**Comment by libauror-a** [comment]
idk i never had this accept when i was trying to get neko working
